README.md

```diff
@@ -4,10 +4,16 @@ A command-line tool to score and rank contributors
 based on their activity over a set of projects
 by analyzing Git histories.
 
+## Requirements
+
+- Git (1.7x or newer)
+- Python >= 3.7
+- GitPython (installed by `pip install GitPython`)
+
 ## Usage
 
 ```bash
-./score-and-rank-contributors [--since since_date] [--authors author_list] project_1 ... project_N
+python score_and_rank_contributors.py [--since since_date] [--authors author_list] project_1 ... project_N
```

Comment:
The original form is easier to use. Any reason to adopt the more verbose form?

In addition, there is a conflict with the main branch. Please rebase this branch to the latest main.

Reply:
There are two primary issues:
1. The hyphen in the filename `score-and-rank-contributor` is not correctly recognized in python module. This means the script cannot be imported directly, leading to complications when testing within the unittest framework.
2. This script requires a third-party dependency, such as `GitPython`, which must be manually installed. Explicitly invoking `python` would make this clearer.